src/sedcli_main.c

```diff
@@ -81,6 +83,14 @@ static cli_option setpw_opts[] = {
 	{0}
 };
 
+static cli_option mbr_control_opts[] = {
+	{'d', "device", "Device node e.g. /dev/nvme0n1", 1, "DEVICE", CLI_OPTION_REQUIRED},
+	{'e', "enable-mbr", "Select MBR Enable column", 0, "FLAG", CLI_OPTION_OPTIONAL_ARG},
```

Comment:
this and following option should take parameter indicating desired value by user vs specifying that value through yet another option